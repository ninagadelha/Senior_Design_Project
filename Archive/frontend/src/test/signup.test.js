import renderer from "react-test-renderer";
import SignUp from "../routes/signup";

jest.mock("react-router-dom");
jest.mock("axios");
jest.mock("../styles/signup.css");
jest.mock("../index.css");

const renderTree = (tree) => renderer.create(tree);
describe("<SignUp>", () => {
  it("should render component", () => {
    expect(renderTree(<SignUp />).toJSON()).toMatchSnapshot();
  });
});
