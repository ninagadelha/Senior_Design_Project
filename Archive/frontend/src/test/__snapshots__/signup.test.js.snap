// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SignUp> should render component 1`] = `
<div>
  <div
    className="split-screen"
  >
    <div
      className="split-left"
    >
      <div
        className="gradient-background"
      >
        <div
          className="logo-container"
        >
          <h1
            className="title"
          >
            MySTEMGrowth Profile
          </h1>
        </div>
      </div>
    </div>
    <div
      className="split-right"
    >
      <div
        className="right-content"
      >
        <div
          className="register-card"
        >
          <h2
            className="card-title"
          >
            Create an Account
          </h2>
          <p>
            Have an account?
             
          </p>
          <div
            className="input-container"
          >
            <p
              className="input-title"
            >
              Email
            </p>
            <input
              className="text-input"
              onChange={[Function]}
              placeholder="Enter your email address"
              type="text"
            />
            <p
              className="error-label"
            />
          </div>
          <div
            className="input-container"
          >
            <p
              className="input-title"
            >
              Password
            </p>
            <input
              className="text-input"
              onChange={[Function]}
              placeholder="Create a password"
              type="password"
            />
            <label
              className="error-label"
            />
          </div>
          <div
            className="input-container"
          >
            <p
              className="input-title"
            >
              Password Re-enter
            </p>
            <input
              className="text-input"
              onChange={[Function]}
              placeholder="Re-enter your password"
              type="password"
            />
            <label
              className="error-label"
            />
          </div>
          <p>
            By creating an account, you agree to our
             
          </p>
          <p
            className="error-message"
          />
          <input
            className="register-button"
            onClick={[Function]}
            type="button"
            value="Register"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
